server/src/main/java/org/opensearch/cluster/metadata/IndexMetadata.java

```diff
@@ -243,6 +243,21 @@ static Setting<Integer> buildNumberOfShardsSetting() {
         Property.IndexScope
     );
 
+    /**
+     * Setting to control the number of search only replicas for an index.
+     * A search only replica has the following properties:
+     * 1. Is not primary eligible
+     * 2.
```

Comment:
The suspense here is killing me. What is property number 2????

Reply:
lol updated the comment